changelog/dep_d1_ops.dd

```diff
@@ -0,0 +1,68 @@
+D1 operator overloads have been deprecated.
+
+The following D1 operator overloads have been deprecated in favor of `opUnary`:
```

Comment:
Since you've done quite a bit of fixing yourself, is in unreasonable to expect that those operators are still widespread ? If so, perhaps a more detailed description is warranted ?

E.g.:
```
* `opNeg` => Affect `-a`. Replace with `opUnary(string op)(...) if (op == "-")` 
```

Reply:
> Since you've done quite a bit of fixing yourself, is in unreasonable to expect that those operators are still widespread ?

Since there were no buildkite failures, I'm under the assumption that the D1 operator overloads are not widespread and are mostly remnants of really old D2 code.  Nevertheless, I'll try to make improvements to the error messages and changelog later today.